tests/models/registry.py

```diff
@@ -111,6 +111,8 @@ class _HfExamplesInfo:
     # QWenLMHeadModel supports multimodal
     "Qwen2ForCausalLM": _HfExamplesInfo("Qwen/Qwen2-7B-Instruct"),
     "Qwen2MoeForCausalLM": _HfExamplesInfo("Qwen/Qwen1.5-MoE-A2.7B-Chat"),
+    "Rwkv6ForCausalLM": _HfExamplesInfo("RWKV/v6-Finch-7B-World3-HF", trust_remote_code=True), # noqa: E501
```

Comment:
Is there any RWKV-6 model with weights in a safetensors format instead of a `pytorch_model.bin`?